ArduPlane/mode_qstabilize.cpp

```diff
@@ -11,29 +11,34 @@ bool ModeQStabilize::_enter()
 
 void ModeQStabilize::update()
 {
-    // set nav_roll and nav_pitch using sticks
+    // fetch normalized roll and pitch inputs
     // Beware that QuadPlane::tailsitter_check_input (called from Plane::read_radio)
     // may alter the control_in values for roll and yaw, but not the corresponding
     // radio_in values. This means that the results for norm_input would not necessarily
     // be correct for tailsitters, so get_control_in() must be used instead.
-    // normalize control_input to [-1,1]
-    const float roll_input = (float)plane.channel_roll->get_control_in() / plane.channel_roll->get_range();
-    const float pitch_input = (float)plane.channel_pitch->get_control_in() / plane.channel_pitch->get_range();
+    float roll_stick = plane.channel_roll->get_control_in();
```

Comment:
I don't think this is better then the above. Previously we got to attach `const` which is a nice, useful tool, and this rearranging removes that. (And doesn't seem particularly clearer)

Reply:
fixed